### DISCUSSION TITLE
Increasing font size on 4k screen

### DISCUSSION THREAD

**Discussion by jehojakim** [vlachoudis/bCNC#1656] [Q&A]
Hi, 

In bCNC 0.9.14-dev, linux py3.8.12, the font of the gui is barely readable by it's tiny size on my 4k screen.
Is there a way to alter it? I tried to find it in all the menu's but so far no luck....

I've seen issue #505 on the subject, but I can't find the menu option suggested there.

regards, jehojakim

**Comment by Gagit41** [comment]
If you are trying to follow issue # 505 or @506 that are related to this post BEWARE the TOOLS tap has been renamed CAM and what you are looking for is now at the left side.  There is a config button there and also below it in the margin is a config that will expand into  Events - Colors - Fonts.  These two config options don't do the same things!!!
![image](https://github.com/vlachoudis/bCNC/assets/134753748/58effa2f-4574-492a-a750-327319aa57ef)
